README.md

```diff
@@ -7,6 +7,12 @@ The model is available from
 [doi:10.5281/zenodo.6221127](https://doi.org/10.5281/zenodo.6221127)
 and released under
 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+The model was created
+by fine-tuning the pre-trained
+[wav2vec2-large-robust](https://huggingface.co/facebook/wav2vec2-large-robust)
+model on
+[MSP-Podcast](https://ecs.utdallas.edu/research/researchlabs/msp-lab/MSP-Podcast.html)
+(v1.7).
```

Comment:
Maybe add " Note that the pre-trained model was pruned from 24 to 12 transformer layers before fine-tuning. "

Reply:
Added "The pre-trained model was pruned from 24 to 12 transformer layers before fine-tuning."